client/public/index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html>
   <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" crossorigin="anonymous">
```

Comment:
curios what this CDN file is

Reply:
I'm using the font-awesome library to get that thumbs up icon for my button. I believe it's just another way to do it instead of how you downloaded the icons locally.